app/models/school.rb

```diff
@@ -2,7 +2,8 @@ class School < ApplicationRecord
   has_paper_trail
   include PgSearch::Model
 
-  belongs_to :responsible_body
+  belongs_to :responsible_body, optional: true
```

Comment:
I don't think we need to make this optional anymore

Reply:
good spot! I've removed this now